### PR TITLE
kflemin3 refactor - move constants from controller to model

### PR DESCRIPTION
**Pull request by kellyfleming**
towards https://github.com/vishnuchalla/expertiza/projects/1#card-79246675

movement of allowed types constants from controller to model

no tests broken by changes, and no new tests required

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,230 +11,231 @@
 #
```
Comment (vishnuchalla) [inline db/schema.rb:11]: Why are seeing changes in this file? when there are no changes in real?

**Review by vishnuchalla — CHANGES_REQUESTED** [review]
Make sure there are no changes in the schema.rb file

File: db/schema.rb (hunk 1 above)
Comment (kellyfleming) [reply db/schema.rb:11]: Honestly, no idea. I cloned your fork and ran the usual setup, including rake db:migrate, and whatever came out of that was what happened with this file. I'll look into it.

**Comment by kellyfleming** [comment]
re-worked on separate, completely new branch.
this closed PR should be ignored

**Pull request closed without merging**